CHANGELOG.md

```diff
@@ -3,6 +3,12 @@
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [[1.2.2](https://github.com/sanger-tol/readmapping/releases/tag/1.2.2)] - [2024-05-23]
```

Comment:
According to the instruction we give, should we add patch word in the release name?
https://pipelines.tol.sanger.ac.uk/docs/contributing/pipeline-release

Reply:
Oh yes, absolutely